knimin/static/vendor/font-awesome/HELP-US-OUT.txt

```diff
@@ -0,0 +1,151 @@
+I hope you love Font Awesome. If you've found it useful, please do me a favor and check out my latest project,
+Fort Awesome (https://fortawesome.com). It makes it easy to put the perfect icons on your website. Choose from our awesome,
+comprehensive icon sets or copy and paste your own.
+
+Please. Check it out.
+
+-Dave Gandy
+
+
+plate list
+
+Ricardo
+
+
+var dict={{ json_encode(dico) }};  
+
+<tr class="spacer"><td></td></tr>
+
+        sql = """SELECT plate_type_id
```

Comment:
Lol. Got confused why they are here. Looks like I used this file as a scratch book... Fixed.